asterius/src/Asterius/Builtins.hs

```diff
@@ -1209,6 +1210,24 @@ barfFunction _ =
     s <- param I64
     callImport "__asterius_barf" [convertUInt64ToFloat64 s]
 
+
+unicodeCBits :: [(AsteriusEntitySymbol, (FunctionImport, Function))]
```

Comment:
This uses `generateRTSWrapper`, but `generateRTSWrapper` treats `i64` in arguments as *signed*, but results as *unsigned*. I expect all arguments/results here to be *unsigned* though; probably leave a comment regarding sign issue here, or we change `generateRTSWrapper` itself (after carefully reviewing the signedness issue for all usages)